packages/apps/dev-wallet/src/modules/wallet/wallet.provider.tsx

```diff
@@ -52,11 +56,14 @@ export const syncAllAccounts = throttle(AccountService.syncAllAccounts, 10000);
 export const WalletProvider: FC<PropsWithChildren> = ({ children }) => {
   const [contextValue, setContextValue] = useState<ExtWalletContextType>({
     networks: [],
+    client: createClient(),
```

Comment:
Shouldn't we pass the hostGenerator based on the active network? As without hosteGenerator we can only work with mainnet and testnet